### ISSUE TITLE
Feature Request: SmallScreen Position Customization

### ISSUE THREAD

**Issue by bryandbor** [LouisBarranqueiro/reapop#32]
On mobile devices, I have wanted to show notifications at the bottom of the screen (to avoid covering navigation), and it would be great to allow the implementation of this library to pick where their "mobile" positioning should be.

I currently implement this change by simply looking for a key of `smallScreenPosition` from the provided `theme` property which must be `POSITIONS.top` or `POSITIONS.bottom`. I have made this change by making a couple small changes to the source code of `NotificationsSystem.js`:

Currently
```
static propTypes = { // line 8
    notifications: React.PropTypes.array.isRequired,
    theme: React.PropTypes.shape({
      smallScreenMin: React.PropTypes.number.isRequired,
      notificationsSystem: React.PropTypes.shape({
        className: React.PropTypes.string
      })
    }).isRequired
  };
```

Modified:
```
static propTypes = {
    notifications: React.PropTypes.array.isRequired,
    theme: React.PropTypes.shape({
      smallScreenMin: React.PropTypes.number.isRequired,
      smallScreenPosition: React.PropTypes.oneOf([
        POSITIONS.top,
        POSITIONS.bottom
      ]), // not marked as required to allow for backwards compatibility
      notificationsSystem: React.PropTypes.shape({
        className: React.PropTypes.string
      })
    }).isRequired
  };
```

Then I simply apply this property (if it exists) when rendering the Notification system for small screens:
Currently:
```
if (windowWidth < theme.smallScreenMin) { // line 67
      return (
        <NotificationsContainer
          key='t'
          position='t'
          theme={theme}
          notifications={notifications}
        />
      );
    }
```

Modified:
```
if (windowWidth < theme.smallScreenMin) {
      return (
        <NotificationsContainer
          key={this.props.theme.smallScreenPosition || POSITIONS.top}
          position={this.props.theme.smallScreenPosition || POSITIONS.top}
          theme={theme}
          notifications={notifications}
        />
      );
    }
```

When I initialize the NotificationsSystem, I simply provide this property (as the current version of the theme does not include this new property):
```
<NotificationsSystem theme={Object.assign(theme, {
  smallScreenPosition: POSITIONS.bottom
})}/>
```

In conclusion, this change simply would allow the implementation to dictate this functionality with no drawbacks. I'm happy to answer questions, if there are any.

**Comment by LouisBarranqueiro** [comment]
Hi @bryandbor , good idea. create a PR, i'm gonna take a look 😄
